utils/swift_build_support/swift_build_support/products/swiftsyntax.py

```diff
@@ -81,9 +84,13 @@ def should_build(self, host_target):
 
     def build(self, host_target):
         if self.args.swiftsyntax_verify_generated_files:
+            script_path = os.path.join(self.source_dir, 'SwiftSyntaxDevUtils')
+
             build_cmd = [
-                os.path.join(self.source_dir, 'build-script.py'),
-                'verify-source-code',
+                os.path.join(self.install_toolchain_path(host_target), "bin", "swift"),
+                'run',
+                '--package-path', script_path,
+                'swift-syntax-dev-utils verify-source-code'
```

Comment:
This needs to be the following to fix the CI failure.

```suggestion
                'swift-syntax-dev-utils', 'verify-source-code',
```

Reply:
somehow this did result in not adding space. 
Don't know why.. 😅

Reply:
There were two issues here, really.

1. You can implicitly concatenate string literals in Python, like you can do in C.
```python
```